## Introduction ##

In a biological context, the cells can be viewed as networks of molecules connected by chemical reactions. The development of massive data collection techniques, as cDNA microarrays and RNA-Seq, allows the simultaneous verification of cell's components estate in multiples instances of time. Computational methods have been extensively used to analyze and to interpret this amount of generated data. In particular, gene regulatory networks (GRN) are used to indicate how the genes are regulated and consequently give insights about the activity of live organisms in molecular level. However, there is an important problem: how to validate the network identification results? This work presents an approach to validate such algorithms by considering the AGN model generation and simulation.
This project provides an Artificial Gene Networks (AGNs) model generation through theoretical models of complex networks, which is used to simulate temporal expression data, which can be used by computational methods to recover the network topology, and then, analyse the results based on complex networks measurements/topology.


---


## Main References: ##

F. M. Lopes, R. M. Cesar-Jr and L. da F. Costa. **Gene expression complex networks: synthesis, identification and analysis**. Journal of Computational Biology, Volume: 18 [Issue 10](https://code.google.com/p/jagn/issues/detail?id=10), page 1353-1367, 2011.
http://dx.doi.org/10.1089/cmb.2010.0118

F. M. Lopes, R. M. Cesar-Jr and L. da F. Costa. **AGN simulation and validation model**. In Advances in Bioinformatics and Computational Biology, Proceedings, volume 5167 of Lecture Notes in Bioinformatics, pages 169â€“173, 2008. Springer-Verlag Berlin.
http://dx.doi.org/10.1007/978-3-540-85557-6_17


## Additional References: ##
F. M. Lopes, E. A. de Oliveira and R. M. Cesar-Jr. **Inference of gene regulatory networks from time series by Tsallis entropy**, BMC Systems Biology, 5 (61), 2011.
http://dx.doi.org/10.1186/1752-0509-5-61

F. M. Lopes, D. C. Martins-Jr, J. Barrera and R. M. Cesar-Jr. **SFFS-MR: a floating search strategy for GRNs inference**.  In: 5th IAPR International Conference on Pattern Recognition in Bioinformatics - PRIB, 2010, Nijmegen, The Netherlands.
http://dx.doi.org/10.1007/978-3-642-16001-1_35

F. M. Lopes, E. A. de Oliveira and R. M. Cesar-Jr. **Analysis of the GRNs inference by using Tsallis entropy and a feature selection approach**.  In: 14th Iberoamerican Congress on Pattern Recognition - CIARP, 2009, Guadalajara, JA, Mexico.
http://dx.doi.org/10.1007/978-3-642-10268-4_55

F. M. Lopes, D. C. Martins-Jr and R. M. Cesar-Jr. **Comparative study of GRNs inference methods based on feature selection by mutual information**.  In: 7th IEEE International Workshop on Genomic Signal Processing and Statistics - GENSIPS, 2009, Minneapolis, MN, USA.
http://dx.doi.org/10.1109/GENSIPS.2009.5174334

F. M. Lopes, D. C. Martins-Jr and R. M. Cesar-Jr. **Feature selection environment for genomic applications**, BMC Bioinformatics, 9 (451), 2008.
http://dx.doi.org/10.1186/1471-2105-9-451


---


## Other works developed by the Creativision and Bioinfo-CP: ##

**DimReduction: Interactive Graphic Environment for Dimensionality Reduction**
http://code.google.com/p/dimreduction/

**Creative Vision Research Group**
http://www.vision.ime.usp.br/~creativision/

**Bioinfo-CP: Bioinformatics and Pattern Recognition Research Group**
http://www.utfpr.edu.br/cornelioprocopio/pesquisa/grupos-de-pesquisa/bioinfo-cp/about-us/about-us